### PR TITLE
feat: dbt plugin

### PR DESCRIPTION
**Pull request by abeizn**
image
image

### CODE REVIEW

File: plugins/dbtplugin/tasks/dbt_task_data.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package tasks
+
+// import (
+// 	"github.com/merico-dev/lake/plugins/helper"
+// )
+
+type DbtOptions struct {
+	SelectedModels string   `json:"selectedModels"`
```
Comment (klesh) [inline plugins/dbtplugin/tasks/dbt_task_data.go:8]: What if `SelectedModels` was omitted? Will this plugin run all models ?

Comment (klesh) [reply plugins/dbtplugin/tasks/dbt_task_data.go:8]: Can we optionaly specify a `modelPath` ?

**Review by klesh — COMMENTED** [review]
Can we support progress?
Maybe by parsing the process `stdout` and extract the information?

File: plugins/dbtplugin/tasks/dbt_converter.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	"github.com/merico-dev/lake/plugins/core"
+	
+)
+
+func DbtConverter(taskCtx core.SubTaskContext) error {
+	data := taskCtx.GetData().(*DbtTaskData)
+	models := data.Options.SelectedModels
+	current_dir, _ := os.Getwd()
+	os.Chdir(current_dir+"/plugins/dbtplugin")
+	cmd := exec.Command("bash", "-c", "dbt run --select " + models)
```
Comment (klesh) [inline plugins/dbtplugin/tasks/dbt_converter.go:15]: exec.Command("dbt","run","--select", models...)

**Review by mindlesscloud — APPROVED** [review]

**Pull request merged**